### PR TITLE
feat(auth/basic): add support for basic auth

### PR DESCRIPTION
**Pull request by ethanfrogers**
adds support for basic auth for users using ldap based authentication

### CODE REVIEW

**Comment by ethanfrogers** [comment]
@jtk54 PTAL :)

File: command/meta.go [hunk 1]
```diff
@@ -216,6 +216,12 @@ func (m *ApiMeta) InitializeClient() (*http.Client, error) {
 			// Misconfigured.
 			return nil, errors.New("Incorrect x509 auth configuration.\nMust specify certPath/keyPath or cert/key pair.")
 		}
+	} else if auth != nil && auth.Enabled && auth.Basic != nil {
```
Comment (jtk54) [inline command/meta.go]: Should use the `isValid()` for basic auth here, currently unused.

Comment (ethanfrogers) [reply command/meta.go]: 👍 fix incoming!

**Comment by ethanfrogers** [comment]
@jtk54 comments addressed! thanks!

**Pull request merged**